src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -533,12 +533,14 @@ bool AsyncRPCOperation_sendmany::main_impl() {
                 std::string hexMemo = smr.memo;
                 zOutputsDeque.pop_front();
 
-                PaymentAddress pa = keyIO.DecodePaymentAddress(address);
-                JSOutput jso = JSOutput(std::get<libzcash::SproutPaymentAddress>(pa), value);
-                if (hexMemo.size() > 0) {
-                    jso.memo = get_memo_from_hex_string(hexMemo);
+                std::optional<PaymentAddress> pa = keyIO.DecodePaymentAddress(address);
+                if (pa.has_value()) {
```

Comment:
This is unnecessary, because `std::get` was previously throwing in the instance where this new condition would be false, so we could just let `.value()` throw. But as this is all going away, meh!